scopes/defender/linter/linter.docs.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-description: Provides a single interface for Component linting
-labels: ['core aspect', 'bit']
+description: Provides a unified interface for the linting of Components of different types
```

Comment:
A unified interface for component linting.

Reply:
done